installing/installing_bare_metal/upi/scaling-a-user-provisioned-cluster-with-the-bare-metal-operator.adoc

```diff
@@ -13,7 +13,15 @@ include::modules/upi-prerequisites-for-scaling-a-upi-cluster.adoc[leveloffset=+2
 include::modules/upi-limitations-for-scaling-a-upi-cluster.adoc[leveloffset=+2]
 include::modules/configuring-a-provisioning-resource-to-scale-user-provisioned-clusters.adoc[leveloffset=+1]
 include::modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc[leveloffset=+1]
-include::modules/upi-managing-existing-hosts-in-a-upi-cluster.adoc[leveloffset=+1]
-include::modules/upi-removing-hosts-from-a-upi-cluster.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster#preparing-the-bare-metal-node_ipi-install-expanding[Preparing the bare metal node]
+
+* xref:../../installing/installing_bare_metal_ipi/ipi-install-installation-workflow#root-device-hints_ipi-install-installation-workflow[Root device hints]
 
+* xref:../../installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster#ipi-install-diagnosing-duplicate-mac-address_ipi-install-expanding[Diagnosing a duplicate MAC address when provisioning a new host in the cluster]
```

Comment:
These all 404

Reply:
Changed links to use new file structure (Bare Metal IPI folder has been removed) as follows, but I'm still getting a 404:
```
* xref:../../installing/installing_bare_metal/ipi/ipi-install-expanding-the-cluster#preparing-the-bare-metal-node_ipi-install-expanding[Preparing the bare metal node]
* xref:../../installing/installing_bare_metal/ipi/ipi-install-installation-workflow#root-device-hints_ipi-install-installation-workflow[Root device hints]
* xref:../../installing/installing_bare_metal/ipi/ipi-install-expanding-the-cluster#ipi-install-diagnosing-duplicate-mac-address_ipi-install-expanding[Diagnosing a duplicate MAC address when provisioning a new host in the cluster]
```

Reply:
I haven't looked at the paths but they are missing their extensions (`.adoc`). Try that first? Might be another issue but that jumps out at me